docs/cli/miactl/20_setup.md

```diff
@@ -85,7 +85,7 @@ sudo mv /tmp/miactl /usr/local/bin
 If you want to run the cli in its environment or you want to test the cli you can use the Docker image:
 
 ```sh
-docker run ghr.io/mia-platform/miactl@0.12.1 miactl
+docker run ghcr.io/mia-platform/miactl:0.12.1 miactl
```

Comment:
Changes to the latest miactl documentation should be made in the original [miactl](https://github.com/mia-platform/miactl) repository since they are automatically migrated here, if applied, this change will be lost on the next miactl documentation update.

Changes to the files versioned docs are ok, since they are frozen files; if you remove this change and open a PR on miactl instead we can merge this one.